lib/daru/dataframe.rb

```diff
@@ -64,6 +64,7 @@ def from_excel path, opts={}, &block
       # Read a database query and returns a Dataset
       #
       # @param dbh [DBI::DatabaseHandle] A DBI connection to be used to run the query
```

Comment:
This should be specified as `@param dbh [DBI::DatabaseHandle, String] A DBI connection to be used to run the query OR Path to a SQlite3 database.`